keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -332,10 +340,17 @@ No.
 
 ### Monitoring Requirements
 
+Considering that this feature is a flag contained fully within in the `kubectl` client, there exist no 
+monitoring requirements for the same.
+
 ###### How can an operator determine if the feature is in use by workloads?
 
 N/A
 
+###### How can someone using this feature know that it is working for their instance?
+
+N/A
```

Comment:
Can I use the kube audit logs to find instances of `kubectl` interacting with sub-resources?  Not sure _why_ I would need to do that, but can I?